cmake-utils/io/fs/CMakeFiles/CMakeDirectoryInformation.cmake

```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
CMAKE generated file

Reply:
Good catch! I updated the PR to remove these files